src/utils/poll/countPollDecryptionFailures.ts

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Relations } from "matrix-js-sdk/src/models/relations";
+
+export const countPollDecryptionFailures = (responseRelations: Relations): number =>
```

Comment:
I cannot find any usage of this in the PR.
Does a poll not already provide the number [here](https://github.com/matrix-org/matrix-js-sdk/pull/3163/files#diff-6d5fabe09478e35a518f33e36f448a7e7c3e224ffa9a4a726fe8ccfeefd7b4f0R101)?